src/Symfony/Component/Validator/Tests/Constraints/FileValidatorTestCase.php

```diff
@@ -459,7 +476,7 @@ public function testUploadedFileError($error, $message, array $params = [], $max
     {
         $file = new UploadedFile(tempnam(sys_get_temp_dir(), 'file-validator-test-'), 'originalName', 'mime', $error);
 
-        $constraint = new File([
+        $constraint = new File(...[
             $message => 'myMessage',
             'maxSize' => $maxSize,
         ]);
```

Comment:
Why using spread operator here? Shouldn't it use named args also?

Reply:
this doesn't work because of `$message` being a variable